tokio/src/runtime/tests/queue.rs

```diff
@@ -101,13 +101,21 @@ fn steal_batch() {
     assert!(local1.pop().is_none());
 }
 
+const fn normal_or_miri(normal: usize, miri: usize) -> usize {
+    if cfg!(miri) {
+        miri
+    } else {
+        normal
+    }
+}
```

Comment:
this is a cute use of `const fn`, i'll have to remember that! :)

nit, take it or leave it: it looks like this is duplicated in a couple of files; we _may_ want to consider making a test-util module? not a hard blocker as it's pretty trivial to write...

Reply:
Eh, for now I think I'll just leave it duplicated.